Update firebase-react-native.md (#6261)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/react-native/destination-plugins/firebase-react-native.md

See [React Native Firebase](https://rnfirebase.io) and [React Native Firebase Analytics](https://rnfirebase.io/analytics/usage) for more details of Firebase packages.

> > info "Installation"

Reviewer: ```suggestion
> info "Firebase dependencies"
```

> > You will need to follow the [install guide]https://rnfirebase.io/#configure-firebase-with-android-credentials for Android and IOS. This includes adding the `GoogleService-Info.plist` and the `google-services.json` file.

Reviewer 2: Might be best to link to each install guide:
```
> info "Installation"
> You will need to follow the install guide for [Android](https://rnfirebase.io/analytics/usage/installation/android) and [iOS](https://rnfirebase.io/analytics/usage/installation/ios). This includes adding the `GoogleService-Info.plist` and the `google-services.json` file.
```

> > You will need to follow the install guide for [Android](https://rnfirebase.io/analytics/usage/installation/android) and [iOS](https://rnfirebase.io/analytics/usage/installation/ios). This includes adding the `GoogleService-Info.plist` and the `google-services.json` file.

Reviewer: ```suggestion
> You will need to follow the install guide for [Android](https://rnfirebase.io/analytics/usage/installation/android){:target="_blank”} or [iOS](https://rnfirebase.io/analytics/usage/installation/ios){:target="_blank”}. This includes adding the `GoogleService-Info.plist` and the `google-services.json` file.
```

Reviewer 2: @forstisabella they _may_ need to do both. React native works for noth android and iOS, so it's possible they would want to do both for each platform

> > You will need to follow the install guide for [Android](https://rnfirebase.io/analytics/usage/installation/android){:target="_blank”} and/or [iOS](https://rnfirebase.io/analytics/usage/installation/ios){:target="_blank”}. This includes adding the `GoogleService-Info.plist` and the `google-services.json` file.

Reviewer: ```suggestion
> To add the dependencies necessary for Segment's Firebase plugin, you'll need to add the `GoogleService-Info.plist` and the `google-services.json` files to your implementation. For more information, follow the instructions in the [Android](https://rnfirebase.io/analytics/usage/installation/android){:target="_blank”} and/or [iOS](https://rnfirebase.io/analytics/usage/installation/ios){:target="_blank”} documentation. 
```
